vector/src/main/java/im/vector/app/features/discovery/DiscoverySettingsController.kt

```diff
@@ -209,18 +210,19 @@ class DiscoverySettingsController @Inject constructor(
             titleResId(R.string.settings_discovery_emails_title)
         }
         when (emails) {
-            is Incomplete -> {
+            Uninitialized,
+            is Loading -> {
```

Comment:
Sadly, even if `is Incomplete` is equivalent to `Uninitialized, is Loading`, the compiler is not clever enough to understand that.